worlds/doom_1993/Options.py

```diff
@@ -0,0 +1,84 @@
+import typing
+
+from Options import AssembleOptions, Choice, Toggle, DeathLink, DefaultOnToggle
+
+
+class Difficulty(Choice):
+    """
+    Choose the difficulty option. Those match DOOM's difficulty options.
+    baby (I'm too young to die.) double ammos, half damage, less monsters or strength.
+    easy (Hey, not too rough.) less monsters or strength.
+    medium (Hurt me plenty.) Default.
+    hard (Ultra-Violence.) More monsters or strength.
+    nightmare (Nightmare!) Monsters attack more rapidly and respawn.
+    """
+    display_name = "Difficulty"
+    option_baby = 0
+    option_easy = 1
+    option_medium = 2
+    option_hard = 3
+    option_nightmare = 4
+    default = 2
+
+
+class RandomMonsters(Choice):
+    """
+    Choose how monsters are randomized.
+    vanilla: No randomization
+    shuffle: Monsters are shuffled within the level
+    random_balanced: Monsters are completely randomized, but balanced based on existing ratio in the level. (Small monsters vs medium vs big)
+    """
+    display_name = "Random Monsters"
+    option_vanilla = 0
+    option_shuffle = 1
+    option_random_balanced = 2
+    default = 1
+
+
```

Comment:
Not sure what items are referring to here. Like, pickups and such? If so, I recommend clarifying that in the RandomItems description.

Reply:
I'll clarify. It's anything but powerups. Powerups are AP Locations.